sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -665,11 +753,13 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     }
   }
 
+  // TODO eventually we want to handle multiple sources of weights
```

Comment:
Yes: in the future there may be multiple producers (GENIE weights, flux weights, etc.). Within a single run of the event weight producer we can mix these calculators arbitrarily, but do want to support factoring this out, generating weights across multiple jobs, etc.

Reply:
Yep. I was too scared to attempt this right now, especially since there is no example file configured that way to test with. I guess we could open an issue to track this.